libs/utils/website/src/lib/auth-link.tsx

```diff
@@ -3,7 +3,11 @@ import {AuthTokenStorageKey} from '@wepublish/authentication/website'
 import {getCookie} from 'cookies-next'
 
 export const authLink = new ApolloLink((operation, forward) => {
-  const {token} = JSON.parse(getCookie(AuthTokenStorageKey)?.toString() ?? '{}')
+  const {token: cookieToken} = JSON.parse(getCookie(AuthTokenStorageKey)?.toString() ?? '{}')
+  const {token: sessionStorageToken} = JSON.parse(
```

Comment:
Add sessionStorage support to it, no need for separate authLink IMO so just baked it into here.